.github/workflows/dietpi-software.bash

```diff
@@ -223,6 +223,7 @@ Process_Software()
 			#207) Moonlight (CLI), "moonlight" command
 			#208) Moonlight (GUI), "moonlight-qt" command
 			209) aCOMMANDS[i]='restic version';;
+			211) aCOMMANDS[i]='hb-service status > /dev/null' aSERVICES[i]='homebridge' aTCP[i]='8581';;
```

Comment:
Not necessary to mute the output. It is actually nice to have it in test results, even if it succeeds:
```suggestion
			211) aCOMMANDS[i]='hb-service status' aSERVICES[i]='homebridge' aTCP[i]='8581';;
```
Indeed this test script and workflow is pretty new and I didn't find time to write some documentation about it yet. Actually worth it since everyone can trigger these tests on the own GitHub repo/fork as well.